services/motion/builtin/builtin_test.go

```diff
@@ -228,24 +227,79 @@ func TestMoveOnMap(t *testing.T) {
 	test.That(t, success, test.ShouldBeTrue)
 }
 
-// TODO(RSDK-2926): Revisit after MoveOnGlobe implementation is completed, needs test cases for optional specs, etc.
 func TestMoveOnGlobe(t *testing.T) {
 	ms, closeFn := setupMotionServiceFromConfig(t, "../data/gps_base.json")
 	defer closeFn()
 
-	success, err := ms.MoveOnGlobe(
-		context.Background(),
-		base.Named("test-base"),
-		geo.NewPoint(0.0, 0.0),
-		math.NaN(),
-		movementsensor.Named("test-gps"),
-		nil,
-		math.NaN(),
-		math.NaN(),
-		nil,
-	)
-	test.That(t, err, test.ShouldBeError, builtin.ErrNotImplemented)
-	test.That(t, success, test.ShouldBeFalse)
+	motionCfg := make(map[string]interface{})
+	motionCfg["motion_profile"] = "position_only"
+	motionCfg["timeout"] = 10.
+
+	t.Run("ensure success to a nearby geo point", func(t *testing.T) {
+		// TODO: This test should be updated such that we can navigate to an
```

Comment:
I think this is fine as is, although it would be an improvement if we used injects to achieve this.  I think we should either do it here or remove the TODO comment.